src/components/Staking/StakingHierarchy.tsx

```diff
@@ -50,7 +50,9 @@ const Container = styled.div`
   }
 `
 
-const Section = styled.div`
+const Section = styled.div<{
+  number: string
```

Comment:
Maybe we can just accept it as a `number` and change how we are using them on the consumers.
```tsx
<Section number={1}>
```

Reply:
I went for the smalles change originally. But yes, this is better.